docs/source/how_to/filter.md

```diff
@@ -0,0 +1,120 @@
+# filter
+
+## What
+
+The purpose of the `filter` command is to provide a easy-to-use graphical-user interface to allow users to:
+
+- **explore** the different AncientMetagenomeDir tables via drop down menus
+- **filter** tables based on different criteria using a standard tabular interface
+- **download** the resulting sequencing data via different methods
+- **download** prepared input samplesheets for pipelines
+- **download** citation information of selected samples
+
+## When
+
+You should use this tool when you wish to find particular types of ancient metagenomic data, but wish to explore the dataset manually and interactively (i.e., don't know exactly what you're looking for yet), and/or if you do not wish to download the AncientMetagenomeDir full tables yourself and filter them within languages such as R or Python (with pandas).
+
+## How
+
+### Loading the interface
+
+Before using `AMDirT filter` you will to make sure you have modern web-browser (Chrome, Firefox, Edge, Safari etc.) available on your machine. This will in most cases be your own laptop or desktop - not a server.
```

Comment:
```suggestion
Before using `AMDirT filter` make sure you have modern web-browser (Chrome, Firefox, Edge, Safari etc.) available on your machine. This will in most cases be your own laptop or desktop - not a server.
```